### PR TITLE
FRONT-2440: Breadcrumb Pattern.

### LINKED ISSUE #14: Refactor Breadcrumb pattern with BCL component

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: templates/overrides/navigation/breadcrumb.html.twig [hunk 1]
```diff
@@ -9,6 +9,15 @@
...
+{% set _breadcrumb_links = {} %}
+{% for breadcrumb_item in breadcrumb %}
+  {% set _links = {
+    'label' : breadcrumb_item.text,
+    'path' : breadcrumb_item.url
+  } %}
+  {% set _breadcrumb_links = _breadcrumb_links|merge([_links]) %}
+{% endfor %}
+
 {% if breadcrumb %}
```
Comment (drishu) [inline templates/overrides/navigation/breadcrumb.html.twig]: _breadcrumb_links

File: templates/overrides/navigation/breadcrumb.html.twig [hunk 2]
```diff
@@ -9,6 +9,15 @@
...
   {{ pattern('breadcrumb', {
     'items': breadcrumb
```
Comment (drishu) [inline templates/overrides/navigation/breadcrumb.html.twig]: _breadcrumb_links

**Review by drishu — CHANGES_REQUESTED** [review]
I made some tests and was thinking to add a Kernel rendering test, but that needs to be discussed with tech leads, so lets implement just this for now.

Please add `- "bcl"` under `file_scan_ignore_directories` in runner.dist.yml until https://github.com/openeuropa/oe_bootstrap_theme/issues/61 is solved.

**Review by Maxfire — DISMISSED** [review]
Reviewed and working

**Review by cmarcmar — APPROVED** [review]

**Review by Maxfire — APPROVED** [review]
approved :+1:

**Review by drishu — APPROVED** [review]

**Pull request merged**